docs/userguide/networkingcontainers.md

```diff
@@ -0,0 +1,224 @@
+
+    $ docker network create -d bridge my-bridge-network
+
+The `-d` flag tells Docker to use the `bridge` driver for the new network. You could have left this flag off as `bridge` is the default value for this flag. Go ahead and list the networks on your machine:
+
+    $ docker network ls
+    NETWORK ID          NAME                DRIVER
+    7b369448dccb        bridge              bridge              
+    615d565d498c        my-bridge-network   bridge              
+    18a2866682b8        none                null                
+    c288470c46f6        host                host
+
+If you inspect the network, you'll find that it has nothing in it.
+
+    $ docker network inspect my-bridge-network
+    [
+      {
+          "name": "my-bridge-network",
+          "id": "615d565d498c771fa43f7d715119304e32d71b3486bd364406ccc981e96599fc",
+          "scope": "local",
+          "driver": "bridge",
+          "ipam": {
+              "driver": "default",
+              "config": [
+                  {}
+              ]
+          },
+          "containers": {},
+          "options": {}
+      }
+    ]
+
+## Add containers to a network
+
+To build web applications that act in concert but do so securely. Networks by definition provide complete isolation for containers.  You can add containers to a network when you first launch a container.
+
+Launch a container running a PostgreSQL database and pass it the `--net=my-bridge-network` flag to connect it to your new network:
+
+    $ docker run -d --net=my-bridge-network --name db training/postgres
+
+If you inspect your `my-bridge-network` you'll see it has a container attached. You can also inspect your container to see where it is connected:
+
+    $ docker inspect --format='{{json .NetworkSettings.Networks}}'  db
+    {"bridge":{"EndpointID":"508b170d56b2ac9e4ef86694b0a76a22dd3df1983404f7321da5649645bf7043","Gateway":"172.17.0.1","IPAddress":"172.17.0.3","IPPrefixLen":16,"IPv6Gateway":"","GlobalIPv6Address":"","GlobalIPv6PrefixLen":0,"MacAddress":"02:42:ac:11:00:02"}}
+
+Now, go ahead and start your by now familiar web application. This time leave off the `-P` flag and also don't specify a network.
+
+    $ docker run -d --name web training/webapp python app.py
+
+Which network is your `web` application running under? Inspect the application and you'll find it is running in the default `bridge` network.
+
+    $ docker inspect --format='{{json .NetworkSettings.Networks}}'  web
+    {"bridge":{"EndpointID":"508b170d56b2ac9e4ef86694b0a76a22dd3df1983404f7321da5649645bf7043","Gateway":"172.17.0.1","IPAddress":"172.17.0.3","IPPrefixLen":16,"IPv6Gateway":"","GlobalIPv6Address":"","GlobalIPv6PrefixLen":0,"MacAddress":"02:42:ac:11:00:02"}}
+
+Then, get the IP address of your `web`
+
+    $ docker inspect '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' web
```

Comment:
I think `'{{ .NetworkSettings.Networks.bridge.IPAddress }}'` can be used here as well, and doesn't rely on order (test it to be sure)